finality-aleph/src/justification/requester.rs

```diff
@@ -72,49 +72,46 @@ impl<B: BlockT> fmt::Display for JustificationRequestStatus<B> {
     }
 }
 
-pub struct BlockRequester<B, RB, C, S, F, V>
+pub struct BlockRequester<B, RB, S, F, V, BB>
```

Comment:
Not sure if it collides with anything, but maybe `BE` is better that `BB` for backend?

Reply:
There are places where we need both, eg `run_validator_node`.